orderer/consensus/etcdraft/blockpuller.go

```diff
@@ -148,9 +148,6 @@ func NewBlockFetcher(support consensus.ConsenterSupport,
 	}
 
 	// TODO: change this to use the new mapping of consenters in the channel config
```

Comment:
what about this TODO?

Reply:
Can you explain, how to use consenters here? Should we take endpoints from consenters rather than configblock?

Reply:
The maxByzantineNodes

Reply:
since it is raft not bft, are we good with maxByzantineNodes=0? in bft we can update maxByzantineNodes from consenters.

Reply:
yep